### DISCUSSION TITLE
experience /w electret capsules

### DISCUSSION THREAD

**Discussion by unosonic** [mcguirepr89/BirdNET-Pi#373] [General]
Since the PUI AOM 5024 have been mentioned here and there, I thought to share my personal experience. I have used about 50 of them in live streaming projects so far, without any problems. One pair in particular is running 24/7 from a rooftop since 1.5 years, protected only by some very cheap windscreen made of some foam and fake fur. It has resisted any kind of weather so far without any problems. Highly recommended for field use! I've also used Primo EM172 and 272 for many years, also mostly without any problems. But the AOMs are are a quarter of the price of the EM and they do a better job in resisting electrical noise, like wifi hotspots very nearby, and other RF dirt. I've used both capsules with various audio recorders as well as with raspberries (0, 2, 3) and various sound cards (IQAudio, Hifiberry, Cirrus/Wolfson). Unfortunately raspberrypi.com have discontinued the great "IQAudio Codec +" board after aquiring IQAudio. A soon as my Pi4 arrives, I'll test whether the "IQAudio zero" board wants to work with it.

**Comment by Klages** [comment]
Nice to hear, I am currently waiting for a 5024 to arrive, and was wondering if a windscreen would be enough to protect it. 
In which direction is your microphone facing?

**Comment by unosonic** [comment]
well, see yourself ;)
This is how it sounds: https://archive.org/details/aporee_52903_64607
![brno_IMG_7530](https://user-images.githubusercontent.com/54538102/173361816-4eb218b5-3d17-46f6-ab65-7a8dba2be12f.JPG)
The windscreen is a piece of open foam, covered with one of these thin socks for testing new shoes, that all wrapped in fake fur from the shop. In one location the mics were once flooded with heavy rain and stopped working (not the ones on the roof!). After drying them out on the heating they continued to work perfectly. I don't spend much effort in order to protect these mics, simply no need.

**Reply by Klages** [reply]
That sounds awesome! Can't wait to get my hands on the better mic's :)
Thank you for the insight.

**Comment by oliverk71** [comment]
I have the AOM-5024 and they are perfectly fine from my point of view. But I did not compare it to other mics. I use deadcats I ordered from Amazon, but artifical fur might be cheaper and fully sufficient. I used hot glue to glue the mic into a plastic tubing, which I mounted next to my lake in the garden, just 40 cm above the ground.

**Comment by DaveBKC** [comment]
I see that you have two of them mounted. Are these feeding the soundcard in stereo or are they wired in parallel?

**Reply by unosonic** [reply]
stereo!

**Comment by nilspupils** [comment]
Hi @unosonic, great you have a lot of experience with the 5024 capsules. I have a specific question you might be able to help with. 
While troubleshooting a noise problem i have found that all tested USB-soundcards provide too little bias voltage to the capsule. Following the datasheet the voltage should be 3v (which is 2.2v at the capsule). It specifies a sensitivity drop of 3db if the voltage is lowered to 2v supply (1.3v at the capsule). None of the tested USB-soundcards provide the recommended level, but are lower than that (0.4-2.0v). I could not test this but i must assume the capsules run with a significant reduction in sensitivity in these settings.
This will be exaggerated even more if capsules are wired in parallel. 
Have you measured the supplied voltage in your installations or have you encountered problems with certain soundcards in this respect? 

Best, Nils

**Reply by nilspupils** [reply]
Ok, i did some more tests: I made a circuit where i could switch between the low (0.4v) bias from the soundcard and a proper 3v from batteries and the output signal was much louder with 3v.
I then lowered the bias voltage continuously until the output signal dropped noticeably, this was roughly at 1v.
I am not sure about real world effects of this but it is something to consider when you want to use the full sensitivity of your microphone. Since two of my soundcards deliver less than 1v (both around 0.4v) it is probably time to retire these. One is a UGREEN device with seperate TRS jacks. 
Best, Nils

**Reply by DaveBKC** [reply]
This very timely for me, as I'm working on my mic as I type. For anyone who has purchased one of these electrets as a finished product, they often have a battery compartment that takes a CR123A 3.0v battery. You don't actually need to retire the sound cards. The datasheet for the  AOM-5024 has a simple schematic for adding 3v power, while blocking it from the input of the sound card. Plug the mic into that (and hopefully power it from the Pi through a 3.3v regulator and a couple of diodes), them plug THAT into the soundcard. 

Assuming I still have a 3.3v regulator laying around, I should be able to breadboard this sometime this week if there's interest.

**Reply by unosonic** [reply]
@nilspupils : Usually lower voltage reduces the max SPL of the microphone, but this most likely won't happen with bird sounds. 
I've just did a measurement on a Pi4, IQAudio soundcard, 2xAOM5024
open circuit (no mic attached): 2.15V
2xAOM5024 connected: 1.5V 
Audible results are quite good, see link above, at least for such a low price tag. Yesterday I've used a stereo setup wich costs 1000 times more, but the results are nor 1000 times better ;)

**Reply by nilspupils** [reply]
@DaveBKC: Thanks for your suggestion, this is exactly what i did when i supplied the capsule with external bias (from a battery). I like the idea of using the 5v from the USB , but I am really curious if it will be possible to use it to generate a bias voltage, as this would have to be extremely clean as it is directly connected to the microphone input (and the usb power is famous for being super noisy). Well worth a try though! Please report back with your findings. 

@unosonic: Thanks very much for your measurements, with 1.5v you seem to be just on the safe side, probably with a little less sensitivity than achievable. I know about the reduction on SPL but i specifically ment a reduction in sensitivity. This is not very clear from the datasheet of the 5024 but is very clear from other datasheets like this one: 
[cmc-6050-40l100.pdf](https://github.com/mcguirepr89/BirdNET-Pi/files/8936126/cmc-6050-40l100.pdf)
I think the same applies to the 5024, even though is not worded very well in the datasheet. Perhaps someone could test it with a better testing setup than i have. 
I would like to try out a better soundcard than the USB ones. Is there one you would recommend?
Best, 
Nils

**Reply by unosonic** [reply]
To be honest, I don't give much on these specs, and rather test them myself under real life conditions. Anyway, the 5024 specs say -3dB when decreasing voltage from 3 to 2V [https://www.puiaudio.com/media/SpecSheet/AOM-5024L-HD-R.pdf](https://www.puiaudio.com/media/SpecSheet/AOM-5024L-HD-R.pdf)
For a high output mic like the AOM it doesn't matter much, unless your preamp runs out of gain. I think both the AOM and Primos are perfect for BirdNet-Pi
About USB-mics: I've used ZoomH2, Olympus LS5, Griffin iMic, Behringer 302 USB-Mixer and a few others I forgot about together with raspberries. They all work in their own flavor. But I definitely prefer audio boards like the Hifiberry DAC+ ADC or IQAudio Zero, both have mic-in with plugin power. Still have to test whether the IQAudio Zero works with a Pi4 (it's made for the Pi Zero WH). The IQAudio Codec + is the best option, low noise, high gain, works on Pi1-4, but unfortunately is out of production since 2 years. 
bests, uno

**Reply by DaveBKC** [reply]
Uno, are you referring to this IQAudio product? https://shop.pimoroni.com/products/iqaudio-codec-zero

**Reply by nilspupils** [reply]
@unosonic: Thanks for your verdict on the issue and i see i probably shouldn't worry about this too much. There will likely be many factors other than sensitivity which matter much more. 
But then again, do you think 0.4v at the capsule is still fine? (Btw. this is just a really crappy implementation of plug-in power with too little current. The open circuit is at around 3v, then it drops to 0.4v when i connect one capsule.) 
Best, 
Nils

**Reply by unosonic** [reply]
@DaveBKC yes, it's this board. I've just tested it, it works fine with Pi Zero/2/3/4 (the ones I have at hand here)
It officially supports only external mono electret microphones, BUT it actually does also stereo in a good quality. There a bit of a weird story behind. If you plan to buy it for stereo use, let me know, I can provide you with the settings. The board is reliable, I have used 20+ of them for 100s of hours of (remote) recordings, e.g.[ this one](https://archive.org/details/rec_mobilemic18_2022_04_30_19_52_05) A friend dropped it at a lake in south Estonia, Pi Zero, IQAudio zero, 2x AOM 5024... Bittern, frogs, singing swans...
@nilspupils I can't tell  if 0.4V is ok, sounds a bit low to me. But it's really simple to make your own PIP with a 9V battery, a resistor and a capacitor, perhaps worth a try
bests, uno

**Reply by nilspupils** [reply]
@unosonic: Beautiful recording! Much better quality than with my USB-interfaces.

**Reply by DaveBKC** [reply]
Thanks @unosonic, I have MicroCenter holding an IQaudio Codec Zero board for me. I'll look you up if/when I start down the stereo path. 👍

**Comment by nilspupils** [comment]
If someone wants to build a "battery box", which supplies the microphone with a higher voltage, this is a great guide:
[Electret Battery Box](https://www.making-sound.co.uk/projects/electret-battery-box.html)

**Comment by nilspupils** [comment]
@unosonic: Can i ask you another question? Have you ever had problems with hum in microphone audio on the raspberry pi? I seem to have some kind of hum in all of my setups and cannot figure it out. Your recording sounded super clean...
Thanks!

**Reply by unosonic** [reply]
hi @nilspupils Yes, especially on the raspberry pi zero / IQAudio zero, together with the Primo EM272. The older EM172 were a bit better, but also a bit problematic. I've experimented with various plugs (plastic, metal, straight, angled, cheap, expensive...) and cables, incl. quality cable like mogami. Best results with Primos I had with cheap plasic angled plugs and a bit of care about how to lay the cable in a box, esp. when using wifi. It's a bit of voodoo HF voodoo.... that said, all this hassle stopped when using the AOM5024 mics. Now I'm using them for all things raspberry, and Primo 272 with my recorders, where they do a good and easy job.

**Comment by nilspupils** [comment]
Thanks, @unosonic! 
I just got better cable (Mogami) and will redo my microphones with it. Will change the plugs also. Can you tell me which ones you used? Will see if that reduces the noise. The funny thing is, on my computer i don't get the noise with the same setup (USB soundcard and mic). It seems the pi makes the system vulnerable for noise somehow. (Many complains about it in the net also).
The 5024 has 2 internal capacitors against interference, probably they are the reason they work better in your setup?  
Just found out you run the Locus Sonus in Kreuzkölln, mine is the BirdNET-Pi in Kreuzberg so we are neighbours!

**Reply by unosonic** [reply]
hi neighbour ;)  sorry I've missed that reply! can I reach you somehow?
I've used very cheap plastic plugs successfully, [like these](https://www.berrybase.de/winkel-klinkenstecker-3-5mm-stereo/3-polig-kunststoffausfuehrung-loetmontage)
